docs/runbooks/VirtControllerRESTErrorsBurst.md

```diff
@@ -2,19 +2,46 @@
 
 ## Meaning
 
-More than 80% of the rest calls failed in virt-controller for the last 5 minutes
+More than 80% of REST calls in `virt-controller` pods failed in the last 5 minutes.
+
+The `virt-controller` has likely fully lost the connection to the API server.
 
 ## Impact
 
-virt-controller has potentially fully lost connectivity to the apiserver. Running workloads will not be impacted but status updates will not be propagated and actions like migrations can not take place.
+Status updates are not propagated and actions like migrations cannot take place. However, running workloads is not impacted. 
 
 ## Diagnosis
 
-There exist two common types of errors:
-- The apiserver is overloaded and we run into timeouts.
-  Issues like this can be identified by checking the apiserver metrics and looking at its response times, overall calls, …
-- The virt-controller pod can’t reach the apiserver. Common issues are: dns issues on the node, networking connectivity issues
+This error is most frequently caused by one of the following problems:
+
+- The API server is overloaded, which causes timeouts. To verify if this is the case, check the metrics of the API server, and view its response times and overall calls.
+
+- The `virt-controller` pod cannot reach the API server. This is commonly caused by DNS issues on the node and networking connectivity issues.
+
+Check whether `virt-controller` can connect to the API server.
+
+1. List all `virt-controller` pods:
+    ```
+     $ export NAMESPACE="$(kubectl get kubevirt -A -o custom-columns="":.metadata.namespace)"
+    ```
+
+1. Obtain the name of the `virt-controller` pod:
+
+    ```
+    $ kubectl get pods -n $NAMESPACE -l=kubevirt.io=virt-controller
+    ```
+
+1. Review the logs for `virt-controller` and check its connectivity to the API server:
+
+    ```
+    $ kubectl logs -n  $NAMESPACE <virt-controller-pod-name>
+    ```
+
 
 ## Mitigation
 
-Check virt-controller logs to identify if it can’t connect to the apiserver at all. If so, delete the pod to force a restart. In this case the issue is normally related to DNS or CNI issues outside of the scope of kubevirt.
\ No newline at end of file
+If the `virt-controller` pod cannot connect to the API server, delete the pod to force a restart:
+
+```
+$ kubectl delete -n <kubevirt-install-namespace> <virt-controller-pod-name>
```

Comment:
Change `<kubevirt-install-namespace>` to `<install-namespace>` because KubeVirt is upstream.